Python/structmember.c

```diff
@@ -4,8 +4,24 @@
 #include "Python.h"
 #include "pycore_abstract.h"      // _PyNumber_Index()
 #include "pycore_long.h"          // _PyLong_IsNegative()
+#include "pycore_object.h"        // _Py_TryIncrefCompare(), FT_ATOMIC_*()
+#include "pycore_critical_section.h"
 
 
+static inline PyObject *
+member_get_object(const char *addr, const char *obj_addr, PyMemberDef *l)
+{
+    PyObject *v = FT_ATOMIC_LOAD_PTR(*(PyObject **) addr);
+    if (v == NULL) {
+        PyObject *obj = (PyObject *) obj_addr;
+        PyTypeObject *tp = Py_TYPE(obj);
+        PyErr_Format(PyExc_AttributeError,
+                     "'%N' object has no attribute '%s'",
+                     tp, l->name);
```

Comment:
Oh, we can actually remove two more lines here :) Feel free to ignore this nit, though.

```suggestion
        PyErr_Format(PyExc_AttributeError,
                     "'%T' object has no attribute '%s'",
                     (PyObject *)obj_addr, l->name);
```

Reply:
getting to know more about internal stuff, so good nit for me :+1: